WIP: jinja templates for prettypfa

Marigold opened the pull request:
WIP, please don't merge. This is only an example of using jinja templates as opposed to raw python string formatting for constructing PrettyPFA. @ludovicc do you think it makes sense or should I stick with string formatting? The benefit of jinja templates is not that clear at this moment, but they could be easier to manage going forward.

ludovicc commented:
I use templates in many other places of the project, it does add one level of abstraction but it can help to bring common code together. Let's go for it

Marigold commented:
Converted all strings to PFA and extracted common code into base templates. Looks much better now 👍 Please don't merge this before we merge https://github.com/LREN-CHUV/python-base-docker-images/pull/13, I still need to manually test it.

Marigold commented:
@ludovicc ready to be merged
